integration/example.go

```diff
@@ -0,0 +1,36 @@
+package integration
+
+import (
+	"crypto/tls"
+	"net/http"
+
+	"github.com/diskordanz/darksky/request"
+	"github.com/shawntoffel/darksky"
```

Comment:
if you use any vendors - you need to add it to glide config `glide get github.com/shawntoffel/darksky`